libenv/sysinfo.c

```diff
@@ -2881,6 +2881,56 @@ static time_t GetBootTimeFromUptimeCommand(time_t now)
 
 /*****************************************************************************/
 
+JsonElement* GetUserInfo(const void *passwd)
```

Comment:
Windows code should use the proper types as well, so ideally both would be type safe. Small thing, I'll change myself and commit.